src/olympia/lib/settings_base.py

```diff
@@ -1100,11 +1100,10 @@ def lazy_langs(languages):
 # Testing responsiveness without rate limits.
 CELERY_WORKER_DISABLE_RATE_LIMITS = True
 
-# Continue serializing in pickle but also accept new JSON format
-# for forwards and backwards compatibility.
-CELERY_ACCEPT_CONTENT = ['pickle', 'json']
-CELERY_TASK_SERIALIZER = 'pickle'
-CELERY_RESULT_SERIALIZER = 'pickle'
+# Only serialize celery tasks using JSON.
+CELERY_ACCEPT_CONTENT = ['json']
```

Comment:
Just a reminder before this get's merged, we should leave `pickle` in the `CELERY_ACCEPT_CONTENT` list for one deployment-cycle to avoid making the deployment harder than it should be. In theory - if I understand this setting correctly - that'll help the worker read old, pickle-serialized tasks but new tasks will be entirely JSON serialized. Theoretically at least… we may want to test this before going live, re-issuing failed tasks is hard.

Reply:
Actually, I think that will break anyway. We have to change the signature of the tasks and how they deal with their arguments...

Reply:
Very fair point, don't worry then. Ops may need to try to keep the queues empty though